developer/code-reviews.md

```diff
@@ -0,0 +1,22 @@
+# Code Review
+
+Code contributions are reviewed by the Reaction core and community contributors.
+All merge requests are reviewed and approved by at least one core contributor before merge.
+
+Here are is a brief checklist of common items reviewed in every merge.
+
+-   Code style guide adhered to and **linted** (`eslint`, 0 errors, no ignores)
+-   i18n keys on all text
+-   i18n values added to appropriate `en.json`
+-   Translations are updated (LingoHub)
```

Comment:
Can we do a quick training session to go over LIngohub? I've looked over the shoulder of aaron doing it, but I don't think I'd know what to do on my own. Maybe after the Dev meeting on Friday or Wednesday?

Reply:
Should probably involve @machikoyasuda and make sure that our deployment docs get updated with any changes to our Lingohub process as well.